documentation/manual/src/main/asciidoc/migration/index.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[ogm-migration]]
+
+== 5.4.2.Final
+
+=== MongoDB
+
+==== MongoDB Java Driver
+
+This patch release updates the underlying MongoDB Java driver from 3.11 to 4.11.  It also fixes some incompatibilities
+with more recent versions of MongoDB that prevent 5.4.1.Final from executing native queries.
+
+As a consequence of the driver update, the oldest MongoDB server release that hibernate-ogm will connect to is now 3.6 (which reached
+end of life in April 2021).
+
+=== MongoDB Server
+
+Recent versions of the MongoDB server have removed support for some features on which hibernate-ogm depend. In particular:
+
+* MongoDB 4.2 removed support for the `$eval` command, on which hibernate-ogm support for stored procedures depends.
```

Comment:
It probably worth mentioning that native queries won't support `maxscan` and `snapshot` anymore: https://github.com/hibernate/hibernate-ogm/pull/1134/files#diff-8b6ff949d6524f98fc432f5eae5b7cec1c828e9ef1281930933b72ef0cc9cdb9L1241